ci-operator/step-registry/openshift-pipelines/ginkgo-install/openshift-pipelines-ginkgo-install-commands.sh

```
@@ -0,0 +1,30 @@
+# Login to the cluster
+if [ -s "${KUBECONFIG:-}" ]; then
+    oc whoami
+else
+    # Login for ROSA & Hypershift platforms
+    (set +x; eval "$(cat "${SHARED_DIR}/api.login")")
+fi
+
+cd /tmp/release-tests-ginkgo
+
+# Install OpenShift Pipelines operator via OLM using the Ginkgo install suite
+(set +x
```

Shellcheck (0.11.0)

[error] Couldn't parse this explicit subshell. Fix to allow more checks.

(SC1073)